.gitlab-ci.yml

```diff
@@ -57,6 +57,15 @@ test_bulk_update:
   tags:
     - ingest-integration-tests
 
+test_direct_archiving:
+  stage: test
+  script:
+     - python -m unittest tests.test_ingest.TestRun.test_direct_archiving
```

Comment:
I would remove the "direct" string from the name.

Reply:
I would leave it for now while we have 2 different way of archiving.